src/azure-cli/azure/cli/command_modules/hdinsight/_help.py

```diff
@@ -55,7 +55,8 @@
 
 helps['hdinsight create'] = """
 type: command
-short-summary: Create a new cluster.
+short-summary: 'Create a new cluster. Incoming BREAKING CHANGE: The default value "large" of parameters
+`--workernode-size` and `--headnode-size` will be removed in the next CLI version.'
```

Comment:
You could use `long-summary` for BREAKING CHANGE part.

Reply:
fixed.